packages/@okta/i18n/dist/properties/login_in.properties

```diff
@@ -1,520 +1,633 @@
-enroll.u2f.instructions = Masukkan Kunci Keamanan Anda ke port USB pada komputer ini. Jika kunci keamanan memiliki tombol atau disk emas, tekan pada tombol atau disk tersebut.
-enroll.u2f.instructionsBluetooth = Jika Anda memakai Kunci Keamanan Bluetooth, tekan tombol.
-enroll.u2f.save = Daftarkan Kunci Keamanan
-enroll.u2f.title = Konfigurasi Kunci Keamanan (U2F)
-enroll.webauthn.instructions = Your browser or device will prompt you to register a Kunci Keamanan or autentikator built-in. Follow the instructions to complete enrollment.
-enroll.webauthn.instructions.edge = Note: If you are enrolling a Kunci Keamanan and Windows Hello or PIN is enabled, you will need to select '<b>Cancel</b>' before continuing.
-enroll.webauthn.save = Enroll
-enroll.webauthn.title = Set up Kunci Keamanan atau autentikator built-in
-enroll.windowsHello.error.notConfiguredHtml = Windows Hello tidak dikonfigurasi. Pilih tombol <b>Mulai</b>, lalu pilih <b>Pengaturan</b> &gt; <b>Akun</b> &gt; <b>Masuk</b> untuk mengonfigurasi Windows Hello.
-enroll.windowsHello.error.notWindows = Windows Hello hanya dapat digunakan di Windows Edge dengan Windows 10. Hubungi admin Anda untuk bantuan.
-enroll.windowsHello.save = Daftar Windows Hello
-enroll.windowsHello.subtitle = Klik di bawah ini untuk mendaftar Windows Hello sebagai formulir pengesahan kedua
-enroll.windowsHello.subtitle.loading = Mohon tunggu, sedang memuat Windows Hello...
-enroll.windowsHello.title = Daftar Windows Hello
-enroll.yubikey.subtitle = Masukkan YubiKey ke port USB dan ketuk untuk membuat kode verifikasi
-enroll.yubikey.title = Setup YubiKey
-error.auth.lockedOut = Akun Anda dikunci. Hubungi administrator Anda.
-error.config = Terdapat kesalahan konfigurasi
-error.enabled.cors = Terdapat kesalahan dalam mengirim permintaan - apakah Anda sudah mengaktifkan CORS?
-error.expired.session = Sesi Anda telah berakhir. Coba untuk login ulang.
-error.invalid.colors.brand = "colors.brand" harus dalam format hex sebanyak enam digit
-error.invalid.identifierFirst = "identifierFirst" hanya bisa diaktifkan jika "idpDiscovery" juga diaktifkan
-error.network.connection = Tidak dapat terhubung ke server. Periksa koneksi jaringan Anda.
-error.oauth.idToken = Terdapat masalah dalam menghasilkan id_token pengguna. Coba lagi.
-error.password.required = Silakan masukkan kata sandi
-error.required.authParams = Tidak ada parameter untuk skema pengesahan yang dikonfigurasi - "OAUTH2"
-error.required.baseUrl = "baseUrl" adalah parameter widget yang diperlukan
-error.required.el = "el" adalah parameter widget yang diperlukan
-error.required.success = Penanganan yang berhasil diperlukan
-error.unsupported.browser = Browser yang tidak didukung
-error.unsupported.cors = Browser yang tidak didukung - tidak ada dukungan CORS
-error.unsupported.localStorage = Browser yang tidak didukung - tidak ada dukungan localStorage
-error.username.required = Silakan masukkan suatu nama pengguna
-errors.E0000001 = Validasi Api gagal: {0}
```

Comment:
this whole file is changing all the strings... seems to me that the new strings here are all in Italian, which is incorrect. If I recall correctly, `in` and `id` both refer to Indonesian, so they probably have the same contents today. If that is true, they should have them after this change too.
If that is not true, then I still have a question of why all the strings are changing here, none is remained unchanged, which seems wrong

Reply:
This was the only manual thing I did. Instead of copying from `id`, I copied from `it`. Fixed it.